lua/wire/client/cl_wirelib.lua

```diff
@@ -162,35 +162,46 @@ function Wire_DrawTracerBeam( ent, beam_num, hilight, beam_length )
 	if (beam_length > 0) then
 
 		local x, y = 0, 0
-		if (ent.GetSkewX and ent.GetSkewY) then
+		if ent.GetTarget then
+			if ent:GetTarget().X ~= 0 or ent:GetTarget().Y ~= 0 or ent:GetTarget().Z ~= 0 then
```

Comment:
These if cases should in included in line 165, otherwise the Skew inputs block won't work.